modules/ding_news/ding_news.features.inc

```diff
@@ -34,16 +34,9 @@ function ding_news_image_default_styles() {
 
   // Exported image style: ding_panorama_list_large_wide.
   $styles['ding_panorama_list_large_wide'] = array(
-    'name' => 'ding_panorama_list_large_wide',
     'label' => 'ding_panorama_list_large_wide',
     'effects' => array(
       22 => array(
-        'label' => 'Manual Crop: Reuse cropped style',
-        'help' => 'Reuse a crop selection from another Manual Crop enabled image style.',
-        'effect callback' => 'manualcrop_reuse_effect',
-        'form callback' => 'manualcrop_reuse_form',
-        'summary theme' => 'manualcrop_reuse_summary',
-        'module' => 'manualcrop',
```

Comment:
Why is this information removed?

Reply:
The new features version removes this, my guess is this is redundant information.

Reply:
Okay